learning.yml

```diff
@@ -0,0 +1,126 @@
+#   database_password, server_password, public_password, ldap_password
+# - Run playbook.
+# - systemctl stop omero-{server,web}
+# - Migrate binary repository to /OMERO/ with rsync.
+# - Migrate database with pg_dump, pg_restore.
+# - Upgrade database.
+#   Do run shape_color_argb_to_rgba.sql.
+#   UPDATE shape SET points = REGEXP_REPLACE(REGEXP_REPLACE(points, '[^\[]*\[([^\]]*).*', '\1'), ', ', ' ', 'g') WHERE discriminator = 'polygon' AND points LIKE 'points[%';
+# - bin/omero db password for root and public users.
+# - systemctl start omero-{server,web}
+
+- hosts: omero-learning
+  roles:
+    - role: ome.lvm-partition
+      lvm_vgname: VolGroup00
+      lvm_lvname: 'var_lib_psql'
+      lvm_lvmount: '/var/lib/pgsql'
+      lvm_lvsize: 20G
+      lvm_lvfilesystem: xfs
+      lvm_shrink: False
+
+    - role: ome.postgresql
+      postgresql_databases:
+        - name: omero
+      postgresql_users:
+        - user: omero
+          password: "{{ database_password }}"
+          databases: [omero]
+      postgresql_version: "10"
+
+    - role: ome.lvm-partition
+      lvm_vgname: VolGroup00
+      lvm_lvname: 'omero'
+      lvm_lvmount: '/OMERO'
+      lvm_lvsize: 150G
+      lvm_lvfilesystem: xfs
+      lvm_shrink: False
+
+    - role: ome.omero-server
+      ice_python_wheel: https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+      omero_server_systemd_start: False
+      omero_server_release: latest
+      omero_server_dbpassword: "{{ database_password }}"
+      omero_server_rootpassword: "{{ server_password }}"
+      omero_server_config_set:
+        omero.client.ui.menu.dropdown.colleagues.enabled: False
+        omero.client.ui.menu.dropdown.everyone.label: "All courses"
+        omero.client.ui.menu.dropdown.leaders.label: "Courses:"
+        omero.db.poolsize: 100
+        omero.jvmcfg.percent.blitz: 50
+        omero.jvmcfg.percent.indexer: 20
+        omero.jvmcfg.percent.pixeldata: 30
+        omero.ldap.config: True
+        omero.ldap.base: "{{ omero_server_ldap_base }}"
+        omero.ldap.username: cn=ldapomero,ou=users,o=dundee
+        omero.ldap.password: "{{ ldap_password }}"
+        omero.ldap.user_filter: "(objectClass=person)"
+        omero.ldap.new_user_group: Virtual Microscope
+        omero.ldap.urls: "{{ omero_server_ldap_urls }}"
+        omero.mail.config: true
+        omero.mail.from: "qa@openmicroscopy.org.uk"
+        omero.mail.host: "smtp.dundee.ac.uk"
+        omero.policy.binary_access: "+read,+write,-image,-plate"
+        omero.security.ignore_case: True
+        omero.security.keyStore: "/etc/pki/java/cacerts"
+        omero.security.trustStore: "/etc/pki/java/cacerts"
+        omero.security.keyStorePassword: "changeit"
+        omero.security.trustStorePassword: "changeit"
+        omero.sessions.timeout: 3600000
+
+    - role: ome.ldap-cert
+      ldap_host: "ldap0.dundee.ac.uk"
+
+    - role: ome.redis
```

Comment:
Can you add this role to `requirements.yml`?

Reply:
Done.